<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormUserAccount</class>
 <widget class="QWidget" name="FormUserAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STEMe - Tạo tài khoản</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../image.qrc">
    <normaloff>:/image/Setup_icon_1.jpg</normaloff>:/image/Setup_icon_1.jpg</iconset>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>480</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_Cancel">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Thoát</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>241</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox_AccountInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:5px</string>
    </property>
    <property name="title">
     <string>Thông tin tài khoản</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Tên tài khoản:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Kiểu tài khoản:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Ngày lập:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_Info_Account">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Info_Type">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Info_Date">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>431</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Thay đổi mật khẩu</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>134</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Mật khẩu cũ:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Mật khẩu mới:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Nhập lại mật khẩu:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>211</width>
       <height>110</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEdit_ChangePassword">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
        <property name="placeholderText">
         <string>&gt; 7 kí tự, không dấu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_ChangeNewPassword">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>&gt; 7 kí tự, không dấu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_ChangeRePassword">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>&gt; 7 kí tự, không dấu</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>211</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton_ChangePassword">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Đổi mật khẩu</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>441</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Xóa tài khoản</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>134</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Chọn tài khoản:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Mật khẩu:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Nhập lại mật khẩu:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>231</width>
       <height>110</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QComboBox" name="comboBox_RemoveAccount">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_RemovePassword">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
        <property name="placeholderText">
         <string>Viết liền, không dấu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_RemoveRePassword">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
        <property name="placeholderText">
         <string>Viết liền, không dấu</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>231</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="pushButton_Remove">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Xóa tài khoản</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>441</width>
     <height>241</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Tạo tài khoản mới</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>134</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Tên tài khoản:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Loại tài khoản:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Mật khẩu:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Nhập lại mật khẩu:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>241</width>
       <height>148</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLineEdit" name="lineEdit_NewAccount">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="placeholderText">
         <string>Viết liền, không dấu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_NewType">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string>Root (Quản lý)</string>
        </property>
        <item>
         <property name="text">
          <string>Root (Quản lý)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Teacher (Giáo viên)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Student (Học sinh)</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_NewPassword">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
        <property name="placeholderText">
         <string>Viết liền, không dấu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_NewRePassword">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
        <property name="placeholderText">
         <string>Viết liền, không dấu</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>241</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="pushButton_ChangePassword_NewCreat">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Tạo tài khoản</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
